File: Week 5/templates/C/C_cd.py


def fast_input():
    import sys
    return sys.stdin.readline().strip()

# Input format: a and b are lists containing n/m elements respectively
def cd(n, m, a, b):
    # Your code to find the maximum number of cds that can be bought goes here
    return -1 # Replace this line with your code


while True:
    n, m = map(int, fast_input().split())
    if n == 0 and m == 0:
        break
    a = []
    b = []
    for i in range(n):
        a.append(int(fast_input()))
    for i in range(m):
        b.append(int(fast_input()))
        
    print(cd(n, m, a, b))

File: Week 5/templates/C/C_cd.cpp
#include <bits/stdc++.h>
using namespace std;

// input format: a and b are vectors of length n/m respectively
int cd(int n, int m, vector<int>& a, vector<int>& b){
    // Your code to find the maximum number of cds that can be bought goes here
}

int main(){
    // Makes cin and cout faster by removing the synchronization between the C and C++ standard streams.
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    int n, m;
    while(cin >> n >> m){
        if(!n && !m){
            break;
        }
        vector<int> a(n), b(m);
        for(int i = 0; i < n; i++){
            cin >> a[i];
        }
        for(int i = 0; i < m; i++){
            cin >> b[i];
        }
        cout << cd(n, m, a, b) << '\n';
    }
}